contracts/FIFOMarket.sol

```diff
@@ -164,6 +158,9 @@ contract FIFOMarket is
   //   return nextRemovalId;
   // }
 
+  /**
+   * @dev Will not work if the removals are for more than one supplier!
```

Comment:
should this be a TODO?

Reply:
I need to refresh my brain but I actually don't think this applies anymore.

Reply:
I'd be curious why not if so